lib/APMTracer.js

```diff
@@ -22,10 +22,9 @@ import { APMSpanContext } from './APMSpanContext';
 import { ConsoleRecorder } from './recorder';
 import { AlwaysSampledSampler } from './sampler';
 
-const CARRIER_PREFIX = 'hawkular-apm-';
-const CARRIER_SPAN_ID = `${CARRIER_PREFIX}spanid`;
-const CARRIER_TRACE_ID = `${CARRIER_PREFIX}traceid`;
-const CARRIER_SAMPLED = `${CARRIER_PREFIX}sampled`;
+const CARRIER_PREFIX = 'HWKAPM';
+const CARRIER_SPAN_ID = `${CARRIER_PREFIX}ID`;
+const CARRIER_TRACE_ID = `${CARRIER_PREFIX}TRACEID`;
 
```

Comment:
At the moment it is only set in the agent, based on the config from the server, but we will need to add mechanisms for it to be set in the opentracing impls - but either way, it needs to be able to propagate a setting that it receives to any services it invokes.